server/src/server.ts

```diff
@@ -97,24 +108,27 @@ interface ISettings {
     yseopml: IServerSettings;
 }
 
-// These are the example settings we defined in the client's package.json
-// file
+// These are the example settings we defined in the client's package.json file
 interface IServerSettings {
-    maxNumberOfProblems: number;
     activateParsingProblemsReporting: boolean;
     pathToPredefinedObjectsXml: string;
+    diagnosticLevel: string;
 }
 
 let engineModel: EngineModel;
-// hold the maxNumberOfProblems setting
 let pathToPredefinedObjectsXml: string;
 let activateParsingProblemsReporting: boolean;
-// The settings have changed. Is send on server activation
-// as well.
+
+// By default, the severity is Information.
```

Comment:
It surprises me to find this default-value-setting in the middle of nowhere. Is it possible to set it directly in the option’s definition, in the `package.json` file?
Also, maybe we should document this default value (tell which value is the default one) in the option’s description or… I dunno.

Reply:
I'll check if that's possible.